### DISCUSSION TITLE
CRITICAL ERROR: transmissionrpc module version 3.0 or higher required

### DISCUSSION THREAD

**Discussion by Al-Azraq** [Flexget/Flexget#3024] [Q&A] [answered]
Hey guys,

I'm not an expert in this but I will try to explain the error I'm getting as best as I can.

I have a Raspberry Pi running Transmission 2.94, and I had Flexget working for a long. However, I had to reinstall everything and I'm trying to get Flexget to work again. I install it, create the config-yml and when testing it I'm getting the error:

transmissionrpc module version 3.0 or higher required

I'm reading that this is due now requiring transmission-rpc but I have no clue on how to solve it.

Could someone ELI5 how to fix this?

Thanks in advance.

**Comment by shumi2008** [comment, marked as answer]
Hello, I am facing exactly the same issue.

Was running flexget v3.1.56, upgraded to the latest version v3.1.129
and encountered the following error every time a transmission task is being executed:

CRITICAL transmission  *****       transmission-rpc module version 3.0 or higher required.

Tried updating with pip install --upgrade transmission-rpc , but no luck even though the version is 3.1.25 I believe.

**Reply by shumi2008** [reply]
UPDATE: installed jail from scratch - seemed to resolve the issue
